### PR TITLE
Support for a config file

### PR DESCRIPTION
**Pull request by dutgcom**
It was a bit tricky to implement, as cri appear to doesn't support config file, so what I'm doing here is checking if there are no options in arguments and if a config file exists - if both are true - I'm inserting to arguments data from a config file. 

This helps to:

1) Still support any number of arguments, like: `rspec_n 100 ./spec/flanky_test_spec.rb ./spec/flanky_test_2_spec.rb` 
2) Still support arguments validation the same way as CRI does - if an incorrect option is specified - user will be notified
3) Project actually doesn't enforce that number of test running time was first, so `rspec_n -c 'some_command' 100` is working and it'll continue to work.

If you have better idea how to do that - please let me know. 

Alternatively I think option can be to switch to OptionParser where it looks like it's possible to parse just a string - https://ruby-doc.org/stdlib-2.7.1/libdoc/optparse/rdoc/OptionParser.html#class-OptionParser-label-Generating+Help 
But I bet there were some reason why CRI was chosen in the first place.

I would've added tests but I see there are no tests at all.
So i've tested it manually separately (with/without file and different formats and errors)
And I'm already using it in a production app.

### CODE REVIEW

**Comment by roberts1000** [comment]
@dutgcom Thanks for submitting this! I'm comfortable with the implementation.

I'm happy to merge the PR in as-is, but I noticed some cleanup opportunities in the code comments and README that we could make before merging. I'm happy to merge the PR as-is, and make the edits. Do you have a preference?

Regarding CRI, there's nothing preventing us from switching to something else down the line.

File: README.md [hunk 1]
```diff
@@ -57,6 +57,20 @@ Or, let the configuration files in the project determine the order:
 
     $ rspec_n 5 --order project
 
+#### Config file
+
+If you create `.rspec_n` file in your project's root, you can put there commands, they'll be used if `rspec_n` run with no options. I.e. `rspec_n 10` will use `.rspec_n` but if you run `rspec_n 10 -c "rm -rf /tmp/* && bundle exec rspect"` - file won't be loaded.
```
Comment (roberts1000) [inline README.md:62]: > If you create a `.rspec_n` file in your project's root, you can put command line options inside it. They'll be used if `rspec_n` is run with no options. (i.e. `rspec_n 10` will use `.rspec_n`, but if you run `rspec_n 10 -c "rm -rf /tmp/* && bundle exec rspec"` the `.rspec_n` file won't be loaded.

File: lib/rspec_n/helpers/arguments_helper.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        options = read_options_from_file
+        if options.any?
+          print "Reading options from `.rspec_n` file\n"
+          return ARGV + options
+        end
+      end
+      ARGV
+    end
+
+    # Read arguments form file only if no arguments provided by user
```
Comment (roberts1000) [inline lib/rspec_n/helpers/arguments_helper.rb:16]: `form` should be `from`.

**Comment by dutgcom** [comment]
Thanks for checking! I'll make edits today and update the PR! :+1:

**Comment by dutgcom** [comment]
@roberts1000 , suggestions were implemented! (I've done force push to my fork so it's still one commit, to keep history nice)

**Pull request merged**